src/app/in-court/hearing-data.service.ts

```diff
@@ -12,10 +12,14 @@ export class HearingDataService {
   public loadHearingDetails(sessionId: string): Observable<any[]> {
     return new Observable<any[]>(observer => {
       this.http.get<any>(`/icp/sessions/${sessionId}`).subscribe(resp => {
-        this.docDataService.getDataFromUrls(resp.documents).subscribe(docs => {
-          observer.next(docs);
-          observer.complete();
-        });
+        observer.next(resp.documents);
+        observer.complete();
+
+        // Old code used to call dm store again.
+        // this.docDataService.getDataFromUrls(resp.documents).subscribe(docs => {
+        //   observer.next(docs);
+        //   observer.complete();
+        // });
       });
```

Comment:
Can we get rid of the commented out code or could it come in handy at a later date?

Reply:
I wasn't sure if it was useful so I kept it... probably can get rid of it though :)